Dockerfile

```
@@ -6,5 +6,11 @@ RUN pip install -r /tmp/requirements.txt
 WORKDIR /opt/iscsc.fr-notify-bot
 COPY . .
 
+# root group already exists, but we need to create the blog_root user
+# Documentation for RUN useradd : https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user
+# Documentation for USER : https://docs.docker.com/engine/reference/builder/#user
+RUN useradd --no-log-init -r -g root blog_root
```

just a slight misspelling
```suggestion
RUN useradd --no-log-init -r -g root blog_bot
```

However, you said you tested it so??

---

I certainly misswrote it in my commit